pkg/template/servicebroker/validation.go

```diff
@@ -17,14 +16,16 @@ func ValidateProvisionRequest(preq *api.ProvisionRequest) field.ErrorList {
 	var allErrs field.ErrorList
 
 	for key := range preq.Parameters {
-		if !templatevalidation.ParameterNameRegexp.MatchString(key) &&
-			key != templateapi.RequesterUsernameParameterKey {
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, this if check shoud be removed
+		if key == templateapi.RequesterUsernameParameterKey {
+			continue
+		}
+
+		if !templatevalidation.ParameterNameRegexp.MatchString(key) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("parameters", key), key, fmt.Sprintf("does not match %v", templatevalidation.ParameterNameRegexp)))
 		}
 	}
 
-	allErrs = append(allErrs, validateParameter(templateapi.RequesterUsernameParameterKey, preq.Parameters[templateapi.RequesterUsernameParameterKey], uservalidation.ValidateUserName)...)
```

Comment:
if it's going away either way(as it is) it's probably not worth worrying too much about where it lives in the short term.